packages/js/src/ui/components/Notification/DefaultNotification.tsx

```diff
@@ -106,17 +121,36 @@ export const DefaultNotification = (props: DefaultNotificationProps) => {
   return (
     <a
       class={style(
-        'notification',
+        SEVERITY_TO_NOTIFICATION_KEYS[props.notification.severity],
```

Comment:
so the notification component will have now a new classes dependent on the severity, for example when it's high:
`nv-notification nv-severityHigh__notification` so the user can use them to adjust the styles, or use the corresponding appearance keys